services/storage-users/README.md

```diff
@@ -173,32 +173,37 @@ ocis storage-users trash-bin purge-expired
 
 The behaviour of the `purge-expired` command can be configured by using the following environment variables.
 
-*   `STORAGE_USERS_PURGE_TRASH_BIN_USER_ID`  
+*   `STORAGE_USERS_PURGE_TRASH_BIN_USER_ID`
```

Comment:
```suggestion
*   `STORAGE_USERS_PURGE_TRASH_BIN_USER_ID`\
```
The double blanks at the end were on purpose as they tell the renderer to put the following text into the next line. Double blanks at the ned can be replaced alternatively with a `\`.